examples/dao/Makefile

```diff
@@ -13,10 +13,8 @@ drop-create-sigma-dao-user: drop-sigma-dao-user create-sigma-dao-user
 # create sigma_dao_user
 create-sigma-dao-user:
 	@echo "Setting up User=$(SIGMA_DAO_UNAME)"
-	@sudo -u postgres createuser $(SIGMA_DAO_UNAME)
-	@sudo -u postgres psql -c "ALTER USER $(SIGMA_DAO_UNAME) WITH ENCRYPTED PASSWORD '$(SIGMA_DAO_PASSWORD)';"
-# grant read only access.
-	@psql -U postgres -d $(POSTGRES_DBNAME) -c "GRANT SELECT ON TABLE $(SIGMA_DAO_OBJECTS) TO $(SIGMA_DAO_UNAME);"
```

Comment:
why we are moving it to setup-sigma-dao?

Reply:
sigma dao relation should be present and indexer should be running to grant access to sigma dao objects. When I was working on it last time, it was available for me. I noticed this deleting the database.